core/core.go

```diff
@@ -471,6 +474,17 @@ func NewTextile(ctx context.Context, conf Config, opts ...Option) (*Textile, err
 		grpcopts = []grpc.ServerOption{
 			grpcm.WithUnaryServerChain(auth.UnaryServerInterceptor(t.noAuthFunc)),
 			grpcm.WithStreamServerChain(auth.StreamServerInterceptor(t.noAuthFunc)),
+			grpc.KeepaliveParams(keepalive.ServerParameters{
+				MaxConnectionIdle:     time.Hour * 24,
+				MaxConnectionAge:      time.Hour * 24,
+				MaxConnectionAgeGrace: time.Hour * 24,
+				Time:                  time.Hour * 2,
+				Timeout:               time.Hour * 2,
+			}),
+			grpc.KeepaliveEnforcementPolicy(keepalive.EnforcementPolicy{
+				MinTime:             time.Minute * 5,
+				PermitWithoutStream: true,
+			}),
```

Comment:
Some gRPC params to ensure our backend is not the cause of the `PROTOCOL ERROR` errors. See https://grpc.github.io/grpc/core/md_doc_keepalive.html